src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -23,6 +23,8 @@ internal static class Utils
         // Keeping this number low will preserve some stack space
         internal const int XmlDsigSearchDepth = 20;
 
+        internal static readonly Encoding DefaultEncoding = new UTF8Encoding(false);
```

Comment:
```suggestion
        internal static readonly Encoding DefaultEncoding { get; } = new UTF8Encoding(false);
```

or change it to an `s_` name

Reply:
no longer relevant since field/property doesn't exist